doc/mf6io/mf6ivar/tex/gwf-rcha-desc.tex

```diff
@@ -35,9 +35,9 @@
 
 \item \texttt{irch}---IRCH is the layer number that defines the layer in each vertical column where recharge is applied. If IRCH is omitted, recharge by default is applied to cells in layer 1.  IRCH can only be used if READASARRAYS is specified in the OPTIONS block.  If IRCH is specified, it must be specified as the first variable in the PERIOD block or MODFLOW will terminate with an error.
 
-\item \texttt{recharge}---is the recharge flux rate ($LT^{-1}$).  This rate is multiplied inside the program by the surface area of the cell to calculate the volumetric recharge rate. The recharge array may be defined by a time-array series (see the "Using Time-Array Series in a Package" section).
+\item \textcolor{blue}{\texttt{recharge}---is the recharge flux rate ($LT^{-1}$).  This rate is multiplied inside the program by the surface area of the cell to calculate the volumetric recharge rate. The recharge array may be defined by a time-array series (see the "Using Time-Array Series in a Package" section).}
```

Comment:
The quotes here are not correct.  They should be latex style.

Reply:
This is a little worrisome, I don't recall touching `gwf-rcha-desc.tex`.  Probably a senior moment. In any event, fixed with 93eadbc